### PR TITLE
Mobile collapsed header

### PR DESCRIPTION
**Pull request by joeyballentine**
Not the best, but it's functional. Opted for a drawer instead of a popover.

image
image

### CODE REVIEW

File: src/elements/header.module.scss [hunk 1]
```diff
@@ -85,3 +90,15 @@
...
+    @media only screen and (max-width: 600px) {
+        display: none;
+    }
+}
+
+.showMobile {
+    @media only screen and (min-width: 600px) {
+        display: none;
+    }
+}
```
Comment (RunDevelopment) [inline src/elements/header.module.scss:104]: `max-width` and `min-width` are both inclusive, so at 600px screen width, both `showMobile` and `hideMobile` will be hidden.

File: src/elements/header.module.scss [hunk 2]
```diff
@@ -85,3 +90,15 @@
...
 }
+
+.hideMobile {
+    @media only screen and (max-width: 600px) {
+        display: none;
+    }
+}
+
+.showMobile {
+    @media only screen and (min-width: 600px) {
```
Comment (joeyballentine) [inline src/elements/header.module.scss:101]:
```suggestion
    @media only screen and (min-width: 601px) {
```

**Comment by RunDevelopment** [comment]
Found some problems:
1. When I open the drawer and then resize the screen to leave mobile mode, the drawer stays open, which is awkward. 
    image
1. In mobile mode, the header's fake height is too small.
    image
1. On doc pages, when I open the mobile menu, I can still interact with the page (which is probably intended) and follow links. Clicking on links does not close the mobile menu. The easiest way to reproduce this is to go to FAQ and then click on the button to "Training". There might be no (easy) way to fix this, so just leave it if you can't find a quick fix.

**Review by RunDevelopment — APPROVED** [review]
Thanks Joey!

**Pull request merged**